Entities/ACrab.cpp

```diff
@@ -2197,233 +2222,319 @@ void ACrab::UpdateAI()
 
 void ACrab::Update()
 {
-    SLICK_PROFILE(0xFF668431);
```

Comment:
This may be a bit of a pain but can you fix it so you don't change all the spacing on this? 1, it's a little messy and we want to always minimize mess like this, and 2. github doesn't have any way that I can find to ignore whitespace changes so I can't tell (without a lot of trouble) what code in here you did or didn't change. If you can't get it fixed lemme know and I'll sort it out by reversing parts of the commit or whatever.